app/assets/stylesheets/common/admin/admin_base.scss

```diff
@@ -12,6 +12,47 @@ $mobile-breakpoint: 700px;
   --space-6: calc(0.25rem * 6);
 }
 
+.admin-site-settings {
+  &__login,
+  &__notifications,
+  &__trust,
+  &__legal,
+  &__font,
+  &__branding,
+  &__navigation,
+  &__security,
+  &__developer,
+  &__rate_limits,
+  &__experimental {
+    .sidebar-section-link.active[data-link-name="admin_all_site_settings"] {
+      background-color: unset;
+      font-weight: unset;
+    }
+    .admin-controls .controls:not(.search),
+    .admin-site-settings-category-nav {
+      display: none;
+    }
+    .admin-site-settings-filter-controls {
+      background-color: var(--secondary);
```

Comment:
It looks like on mobile, some of the filter element is showing up unexpectedly. I couldn't figure out where the color was coming from, but it seems like setting the margin-bottom to unset does the trick.

![image](https://github.com/user-attachments/assets/669f2457-a79a-46a9-a8d6-fbfde15cbd88)